spec/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Companies House - Limited Partnerships API",
+    "description": "An API to allow to the registration, transition and update of limited partnerships",
+    "version": "1.0.0"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "tags": [
+    {
+      "name": "Limited Partnerships"
+    }
+  ],
+  "paths": {
+    "/limited-partnerships": {
```

Comment:
The controller POST end-point currently seems to map to just the root path ("/") so is this right, having "/limited-partnerships"?

Reply:
This is not the create endpoint it is just the health check. I was following what we did for roe based on the advice on the ticket, we don't tend to put health checks in the spec and it needed some endpoint included in it.

Reply:
Ah right, I see.

But... is the health-check end-point working on that URL? For ROE we added [this](https://github.com/companieshouse/overseas-entities-api/blob/main/src/main/resources/application.properties) config so wondering if similar is needed here too?

Reply:
![Screenshot 2024-10-24 at 09 10 27 (2)](https://github.com/user-attachments/assets/cec824d2-e129-48ab-abaa-8b9f8e6c17c4)

The endpoint does work, surprised this has it's own config though. I can add something in.

Reply:
Yes, so that's the end-point you added to the controller but I don't think the health-check end-point will be working - that requires actuator config which is missing. We can have a call.

Reply:
Updated to use the actuator by adding application.properties config.